gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigResourceLocalHandler.java

```diff
@@ -109,8 +109,12 @@ public FlowConfig getFlowConfig(FlowId flowId) throws FlowConfigLoggedException
   public CreateResponse createFlowConfig(FlowConfig flowConfig, boolean triggerListener) throws FlowConfigLoggedException {
```

Comment:
Please document the 409/201 behavior to make it clear that scheduled and run-once jobs are treated differently.